src/renderers/shared/fiber/ReactFiberBeginWork.js

```diff
@@ -176,7 +183,9 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>, s
     }
     // Rerender
     const instance = workInProgress.stateNode;
+    ReactCurrentOwner.current = workInProgress;
     const nextChildren = instance.render();
+    ReactCurrentOwner.current = null;
```

Comment:
Or you can just add it at the end of performWork without the try/finally so that we can land this and get some test scores up! :)